infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/devfile/ContainerSearch.java

```diff
@@ -102,7 +102,8 @@ private Stream<Container> findContainers(HasMetadata o) {
     } else if (o instanceof DeploymentConfig) {
       return ((DeploymentConfig) o).getSpec().getTemplate().getSpec().getContainers().stream();
     } else if (o instanceof Template) {
-      return ((Template) o).getObjects().stream().flatMap(this::findContainers);
+      return ((Template) o)
+          .getObjects().stream().flatMap((Object t) -> findContainers(((HasMetadata) t)));
```

Comment:
@vinokurig is this change relevant to this PR?

Reply:
should be, in kubernetes client 6 Template objects were of HasMetadata type. Now they are untyped (Object), so cast is needed.